test/nativepath/NativesOK.java

```diff
@@ -33,11 +33,13 @@ public class NativesOK {
     public static void main(String[] args) {
         String j_l_path = System.getProperty("java.library.path");
         String t_native = System.getProperty("test.nativepath");
+        String c_native = System.getProperty("correct.nativepath").replace("/", File.separator);
```

Comment:
the name "correct.nativepath" is slightly off, since it needs to be fixed up, but well, ... it works!